cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/MaintenancePlan.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ * Copyright 2021 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
```

Comment:
I suspect this was updated by mistake instead of https://github.com/linkedin/cruise-control/pull/1515/files#r613595943?